qt/scientific_interfaces/ISISReflectometry/GUI/Experiment/QtExperimentView.cpp

```diff
@@ -95,19 +111,51 @@ void QtExperimentView::initLayout(const Mantid::API::IAlgorithm_sptr &algorithmF
   connect(m_ui.addPerAngleOptionsButton, SIGNAL(clicked()), this, SLOT(onNewLookupRowRequested()));
 }
 
+/** Set a column tooltip from a map, if it exists
+ *
+ * @param column : the column index
+ * @param tooltips : a map of column index to tooltip
+ * @return : true if the tooltip was set, false if not found
+ */
+bool QtExperimentView::setTooltipFromMap(int column, std::map<int, std::string> tooltips) {
+  auto tooltipIt = tooltips.find(column);
+  if (tooltipIt == tooltips.end()) {
```

Comment:
`if (auto tooltipIt = tooltips.find(column);  tooltipIt == tooltips.end())`

Reply:
I'd normally agree but because we need two statements inside the `if` I'm not sure it adds much value and in some ways is more confusing...